doc/wg/core/notes/core-notes-2020-09-25.md

```diff
@@ -0,0 +1,159 @@
+# Tock Core Notes 2020-09-25
+
+## Attending
+ * Pat Pannuto
+ * Johnathan Van Why
+ * Philip Levis
+ * Amit Levy
+ * Leon Schuermann
+ * Alistair
+ * Brad Campbell
+ * Hudson Ayers
+ * Branden Ghena
+ * Samuel Jero
+ * Vadim Sukhomlinov
+
+## Updates
+ * Phil: Timer merged!!
+ * Brad: elf2tab rounds application outputs to power-of-2, because of Cortex-M MPU limitations. For Risc-V, this power-of-2 isn't a requirement, so looking at moving the logic out of elf2tab
+     * Amit: Sounds reasonable, perhaps tockloader responsibility?
+     * Brad: Actually, it is already a tockloader responsibility. The multiple compiled binaries already a major R5 complexity
+     * Amit: Yeah, seems like a pre-tockloader choice whose time has come to move
+ * Johnathan: OpenSK is working on adding NFC support for 52840; also working on CryptoCell, but that's moving slower b/c very complex
+
+## Interrupt control semantics in Tock/HIL
+ * A: Summary: Who in the kernel is responsible to enable/disable NVIC on ARM / equiv on R5?
+ * P: Not just about who, but about what happens
```

Comment:
I think we need to be a bit more clear that "A" is Amit and "P" is Phil.

Reply:
Yeah, maybe do a find/replace on all of these name abbreviations

Reply:
Expanded out all the names ✔️